app/src/main/java/org/eyeseetea/malariacare/data/remote/PullDhisSDKDataSource.java

```diff
@@ -172,7 +172,7 @@ private boolean isNetworkAvailable() {
         return netInfo != null && netInfo.isConnectedOrConnecting();
     }
 
-    public void wipeDataBase() {
+    public static void wipeDataBase() {
```

Comment:
@idelcano not is best practice in this case convert to static this method because PullDhisSDKDataSource is a class with state. Better refactor extract this method to other place.